lua/nvim-treesitter/textobjects/select.lua

```diff
@@ -203,7 +203,11 @@ function M.detach(bufnr)
       query = query.query
     end
     if query then
-      vim.keymap.del({ "o", "x" }, mapping, { buffer = bufnr })
+      for _, mode in ipairs { "o", "x" } do
+        if vim.fn.mapcheck(mapping, mode, false) ~= "" then
+          vim.keymap.del(mode, mapping, { buffer = bufnr })
+        end
+      end
```

Comment:
any idea why this happens? the checks to set the mappings and to unset them are symmetric. Unless something changed in the meantime (e.g. config change) this shouldn't cause a problem.

But still better to check before the unset 👍

Reply:
I actually couldn't reproduce it in my settings, but maybe when the setup has the wrong keymap (like `a*` or `a/`. I don't know if they are allowed) it isn't set at all to begin with? Also I noticed that the setting contains non-existing queries